alea/models/blueice_extended_model.py

```diff
@@ -84,17 +87,18 @@
         because it calls the ll and requires data to be set.
         We should update this function in the future after we stop using blueice.
         """
+        generate_values = self.parameters(**kwargs)  # kwarg or nominal value
         ret = dict()
 
         # calling ll need data to be set
         self_copy = deepcopy(self)
         self_copy.data = self_copy.generate_data()
 
         # ancillary likelihood does not contribute
-        for ll_term in self_copy._likelihood.likelihood_list[:-1]:
-            ll_pars = list(ll_term.rate_parameters.keys()) + list(ll_term.shape_parameters.keys())
-            ll_pars += ["livetime_days"]
-            call_args = {k: i for k, i in kwargs.items() if k in ll_pars}
+        # TODO: Make a self.likelihood_temrs dict with the likelihood names as keys and the corresponding likelihood terms as values.
+        for ll_term, parameter_names in zip(self_copy._likelihood.likelihood_list[:-1],
+                                            self_copy._likelihood.likelihood_parameters):  # ancillary likelihood does not contribute
```

Comment:
In this case and other similar cases, the comment should be put in another line, and the indent should be changed, according to https://legacy.python.org/dev/peps/pep-0008/#indentation. 

```
        # ancillary likelihood does not contribute
        for ll_term, parameter_names in zip(
            self_copy._likelihood.likelihood_list[:-1],
            self_copy._likelihood.likelihood_parameters):
```

Reply:
Now that we're talking indents: I think we should in this case do one more indent otherwise we'll get "continuation line with same indent as next logical line (E125)".. And this makes code terrible to read.

Reply:
Ah ok nice it even makes this suggestion in the ref you sent below 😄